packages/apiserver/src/resources/materials/material.service.ts

```diff
@@ -158,4 +126,362 @@ export class MaterialService {
       return material;
     });
   }
+
+  private buildSearchQuery(
+    searchInput: SearchInput,
+    selectHidden: boolean
+  ): SelectQueryBuilder<Material> {
+    const queryBuilder = Material.createQueryBuilder('material').innerJoin(
+      'status',
+      'currentStatus',
+      'material.current_status_id = currentStatus.id'
+    );
+    let isFirstWhere = true;
```

Comment:
can we omit `isFirstWhere` variable? I mean can we start our query with `andWhere` without need to write `where`. I am currenlty watching at `status.service` column 16, there are query builder that starts with `andWhere`

Reply:
Didn't know you can do it.
Sure, will fix

Reply:
I also don't know just saying that I saw smth like that in our codebase)

Reply:
It works.
Fix is pushed